src/tests/utils/definitions/no-auth.json

```diff
@@ -0,0 +1,85 @@
+{
+  "actions": {
+    "create_recipe": {
+      "_dump_version": 1,
+      "custom_fields_result_url": "https://wt-c396b46e7e285c63f4bd6d4f8d32dc2e-0.run.webtask.io/fields-output",
+      "custom_fields_url": "https://wt-c396b46e7e285c63f4bd6d4f8d32dc2e-0.run.webtask.io/fields-input",
+      "fields": {
+        "name": {
+          "choices": null,
+          "default": null,
+          "help_text": null,
+          "label": "Name",
+          "list": false,
+          "parent_key": null,
+          "placeholder": null,
+          "prefill": null,
+          "required": true,
+          "searchfill": null,
+          "send_in_json": true,
+          "sort": null,
+          "type_of": "Unicode"
+        }
+      },
+      "help_text": "Creates a new Recipe.",
+      "hide": false,
+      "important": true,
+      "label": "Create Recipe",
+      "noun": "Recipe",
+      "sample_result_fields": [
+        {
+          "key": "authorId",
+          "type": "int"
+        },
+        {
+          "key": "id",
+          "type": "int"
+        },
+        {
+          "important": true,
+          "key": "name",
+          "type": "unicode"
+        }
+      ],
+      "url": "https://wt-c396b46e7e285c63f4bd6d4f8d32dc2e-0.run.webtask.io/recipe-create"
+    }
+  },
+  "auth_fields": {},
+  "general": {
+    "auth_data": {},
+    "auth_label": "",
+    "auth_mapping": {},
+    "auth_type": "Unknown Auth",
```

Comment:
I think we should try to render a `custom` auth for "Unknown Auth". No Auth would be [`No Auth Needed`](https://github.com/zapier/zapier/blob/develop/developer/models.py#L300).

Reply:
I made an app https://zapier.com/developer/builder/app/79939/development and specified "No Auth" for the authentication. The dumped result (https://zapier.com/api/developer/v1/apps/79939/dump) is `"auth_type": "Unknown Auth"`. Am I missing something here?

Reply:
Well, in that case let it be, then!